plugins/orchestrator/src/components/WorkflownstancesViewerPage/AssessmentResultViewer.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useMemo } from 'react';
+
+import { InfoCard } from '@backstage/core-components';
+
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Chip,
+  Link,
+  Typography,
+} from '@material-ui/core';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+
+interface AssessmentResultViewerProps {
+  result: Record<string, unknown> | string | undefined;
+}
+
+interface WorkflowOption {
+  id: string;
+  name: string;
+}
+
+export const AssessmentResultViewer = (props: AssessmentResultViewerProps) => {
+  const { result } = props;
+
+  const jsonSource = useMemo(() => {
+    if (!result) {
+      return undefined;
+    }
+    if (typeof result === 'string') {
+      return JSON.parse(result);
+    }
+    return result;
+  }, [result]);
+
+  const idToUrl = (id: string) => {
+    return `/orchestrator/workflows/${id}/execute`;
```

Comment:
We should use the `executeWorkflowRouteRef` from the `routes.ts` file instead of hardcoding the route.
There are some examples of how to use these refs throughout the code.

Reply:
Fixed.